### ISSUE TITLE
Tag releases

### ISSUE THREAD

**Issue by cromwellryan** [sparkbox/mina-circle#34]
Should we be tagging releases?

**Comment by yock** [comment]
Yes! I'm in the middle of a big-ish refactor now. That would be a good opportunity to start tagging with some decent release notes.

**Comment by cromwellryan** [comment]
Mind if I tag the last two versions? I was looking to see the diff on 1.2.1 and 1.2.2. Not a huge deal, but ¯\_(ツ)_/¯

**Comment by yock** [comment]
Yeah, that'd be great.
